manifests/init.pp

```diff
@@ -100,7 +100,7 @@
   String $rvm_ruby = $katello_devel::params::rvm_ruby,
   String $rvm_branch = $katello_devel::params::rvm_branch,
   Boolean $use_scl_ruby = $katello_devel::params::use_scl_ruby,
```

Comment:
I do wonder what's the use of this if `$scl_ruby` can be undef. Perhaps it should be dropped instead by changing this part:
https://github.com/theforeman/puppet-katello_devel/blob/230b6165953b079c84c4396f86269fea60db8e7b/manifests/bundle.pp#L22
Make it `if $scl_ruby` instead.